tests/js/browser-binder-tests.js

```diff
@@ -0,0 +1,639 @@
+// Test "binder" components using `gpii-test-browser`.
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-test-browser");
+gpii.tests.browser.loadTestingSupport();
+
+var path    = require("path");
+var url     = require("url");
+var fileUrl = url.resolve("file://", path.resolve(__dirname, "../static/tests-binder.html"));
+
+fluid.registerNamespace("gpii.binder.tests");
+
+// TODO:  If we find ourselves making model changes often, move this to `gpii-test-browser` and polish it up for wider use.
+gpii.binder.tests.applyChange = function (componentPath, modelPath, newValue) {
+    var component = fluid.getGlobalValue(componentPath);
+    component.applier.change(modelPath, newValue);
+};
+
+
+fluid.defaults("gpii.binder.tests.caseHolder", {
```

Comment:
I started working on the conversion today, but could not find any equivalent to Nightmare's `type` function in the tests for `infusion`.  @jobara or @amb26, do you have any examples of simulating typing into a text input or textarea?  All the tests I see in `infusion` change the `value` of the element, as demonstrated here:

https://github.com/fluid-project/infusion/blob/master/tests/component-tests/inlineEdit/js/InlineEditTests.js#L143

The existing Nightmare tests follow the browser behavior of sending individual keys to the field, and a `change` event is fired once the field loses focus.  This is what I need to test.    Is there an existing approach for this with `jquery.simulate`?